tests/conftest.py

```diff
@@ -41,6 +41,38 @@ def _fixture_code(entry_point_name):
     return _fixture_code
 
 
+@pytest.yield_fixture
+def fixture_remotedata(fixture_localhost, shared_datadir):
+    """
+    Return a `RemoteData` with contents from the specified directory. Optionally a
+    mapping of strings to replace in the filenames can be passed. Note that the order
+    of replacement is not guaranteed.
+    
+    The RemoteData node is yielded and points to a folder in /tmp, and is removed at the end
+    """
+    from aiida.orm import RemoteData
+    from aiida.common.folders import SandboxFolder
+
+    replacement_mapping = {'gaas': 'aiida'}
+    dir_path = str(
+        shared_datadir / 'gaas'
+    )  # TODO: Remove cast to 'str' when Python2 support is dropped.
+
+    with SandboxFolder() as sandbox:
```

Comment:
This was my choice. The reason is that `tempfile.TemporaryDirectory` is not there in python 2.7 and I didn't want to implement a back port for it (of course if you know of one already, ok to use it). If you want I can open an issue to replace it with TemporaryDirectory as soon as we drop py2.

Reply:
Ah, I see - that makes sense. It would probably be good to put a comment there to explain it, and sowe can search for `python2` for things like this once we drop support.

Reply:
Added.